### DISCUSSION TITLE
Insufficient Permissions

### DISCUSSION THREAD

**Discussion by cassiano-monuv** [easy-global-market/nifi-file-identity-provider-bundle#68] [Q&A]
I made the settings as described in the readme, but the users are without permission. How does it work to define the admin user?

![image](https://github.com/user-attachments/assets/a75cfc2e-9dd8-4026-bc97-1ab67e28cefc)

**Comment by bobeal** [comment]
Hi @cassiano-monuv,

The initial admin user is generated by NiFi when it starts (see `single-user-provider` section in file `conf/login-identity-providers.xml`). You can use it to log in the first time and give rights to users created with https://github.com/easy-global-market/nifi-file-identity-provider-bundle#initialize-user-credentials.

What should also work is to change the id of the Initial User / Admin Identity in file `conf/authorizers.xml` and set it to one of the user you have just created (in the three places). 

Let me know.

**Comment by cassiano-monuv** [comment]
@bobeal 
I tried to configure the settings as you mentioned, but I keep getting an error. The initial admin user generated by nifi loses access and the credentials are invalid. No access is granted to any user. I will share my configuration files, could you help me find what I'm doing wrong?

**structure of my project:**

- nifi
-- DockerFile
-- authorizers.xml
-- login-credentials.xml
-- login-identity-provideres.xml
-- nifi.properties
-- nifi-file-identity-provider-nar-1.27.0.nar
-- docker-compose.yml

**Download dos arquivos:** https://wormhole.app/vA9AL#M-Ib6ftkzLttCiplPTnOFQ

**Reply by bobeal** [reply]
Hi @cassiano-monuv,

I made some minor improvements in the README to make it easier to configure the extension. I have done the full configuration from a clear NiFi installation and it worked well.

I had a look at the files you provided. I just noticed that in `authorizers.xml`, it is missing the declaration of the initial admin (https://github.com/easy-global-market/nifi-file-identity-provider-bundle/tree/master#create-and-give-admin-rights-to-the-first-user), it also seems this file is not mounted into your Docker image.

Regards,

Benoit.
